template/template.go

```diff
@@ -76,3 +84,137 @@ func (t *Template) ExecuteHTMLString(html string, data interface{}) (string, err
 	err = tmpl.Execute(&buf, data)
 	return buf.String(), err
 }
+
+type FuncMap map[string]interface{}
+
+var DefaultFuncs = FuncMap{
+	"upper": strings.ToUpper,
```

Comment:
These names are problematic as they don't follow the Go names for these functions. What we should probably do is rename `title` to `toTitle` over in Prometheus.